src/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -60,6 +60,15 @@ export function VisualizeListingController($injector, $scope) {
     selectedItems = this.pageOfItems.slice(0);
   };
 
+  this.isAscending = () => sortProperties.isAscending();
```

Comment:
I think it's worth future-proofing this a bit by changing it so that it accepts a property name and returns the ascending value of that SortProperty. The only reason this currently works from a UI perspective is because the sort arrow gets hidden if a column isn't being sorted on. If we ever decide to show that arrow for columns that aren't being sorted on (e.g. maybe we show it on hover), then the direction would be incorrect.